build/ci.go

```diff
@@ -173,6 +173,21 @@ func doInstall(cmdline []string) {
 	if flag.NArg() > 0 {
 		packages = flag.Args()
 	}
+
+	if runtime.GOOS == "windows" {
+		// Resolve ./... manually and remove vendor/bazil/fuse (fuse is not in windows)
+		out, err := goTool("list", "./...").CombinedOutput()
+		if err != nil {
+			log.Fatalf("package listing failed: %v\n%s", err, string(out))
+		}
+		packages = []string{}
+		for _, line := range strings.Split(string(out), "\n") {
+			if !strings.Contains(line, "bazil") {
+				packages = append(packages, strings.TrimSpace(line))
+			}
+		}
+	}
```

Comment:
We should not remove for all platforms since "vendor/basil.org/fuse" is required by other platforms for the test cases to work.
Fuse support is not there only in windows.. thats why i removed those for windows.